app/views/welcome/tasks.html.erb

```diff
@@ -0,0 +1 @@
+<h1 class="tasks-heading">My Tasks</h1>
```

Comment:
I don't see a route for this. It might be good to keep in mind that tasks won't necessarily be on the welcome controller. Where do you think we could see a listing of all tasks to be done?